DOCUMENTATION.md

```diff
@@ -25,6 +25,26 @@ client = Notifications::Client.new(api_key)
 
 To get an API key, [sign in to GOV.UK Notify](https://www.notifications.service.gov.uk/sign-in) and go to the __API integration__ page. You can find more information in the [API keys](#api-keys) section of this documentation.
 
+## Error messages
+
+Error messages consist of:
+
+- a status_code, for example ‘400’
+- an error, for example’BadRequestError’
```

Comment:
```suggestion
- an error, for example ’BadRequestError’
```

this typo this present in all PRs btw :)

Reply:
Thanks @samuelhwilliams. I'm working on these tomorrow so will pick up the errors then :)